<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="ln_say1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_say2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1.Sayı:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2.Sayı:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ln_sonuc">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SONUÇ:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>241</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>240</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosya nın içindekiler</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dosya ya yaz</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_lemler">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>Process</string>
    </property>
    <addaction name="actionAddition"/>
    <addaction name="action_Subtraction"/>
    <addaction name="action_multiplication"/>
    <addaction name="actionDivision"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_lemler"/>
   <addaction name="menuOperation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="actionSave"/>
  </widget>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resimler/images/copy.png</normaloff>:/Resimler/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resimler/images/cut.png</normaloff>:/Resimler/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resimler/images/paste.png</normaloff>:/Resimler/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resimler/images/new.png</normaloff>:/Resimler/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Resimler/images/save.png</normaloff>:/Resimler/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAddition">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Process/process_images/Toplama.png</normaloff>:/Process/process_images/Toplama.png</iconset>
   </property>
   <property name="text">
    <string>Addition</string>
   </property>
  </action>
  <action name="action_Subtraction">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Process/process_images/Cikarma.png</normaloff>:/Process/process_images/Cikarma.png</iconset>
   </property>
   <property name="text">
    <string>Subtraction</string>
   </property>
  </action>
  <action name="action_multiplication">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Process/process_images/Carpma.png</normaloff>:/Process/process_images/Carpma.png</iconset>
   </property>
   <property name="text">
    <string>multiplication</string>
   </property>
  </action>
  <action name="actionDivision">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/Process/process_images/Bolme.png</normaloff>:/Process/process_images/Bolme.png</iconset>
   </property>
   <property name="text">
    <string>Division</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
